### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.4] message-toast content not able to read by screen reader

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11287]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue:https://sap.github.io/fundamental-ngx/#/core/message-toast
Screen reader not able to read any message returned by Message Toast popup. User should be informed about all the data present/appear on the screen after performing any action.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use message toast with some message. Screen reader will not able to read any message returned by Message Toast popup. 

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

Yes, I tested and I can confirmed that they are not announced - Win 11, Jaws 2023 and Edge.
Looking at the code I think you need to switch the implementation, as you rely on dynamically adding a live region on the fly. And typically those work differently, you need to have a rendered element with aria-live polite and when you change the text then the value is announced.
The toast in UI5 has a role "alert": https://ui5.sap.com/#/entity/sap.m.MessageToast/sample/sap.m.sample.MessageToast

You can switch the implementation and I assume it will be announced ok.

Best Regards,
Nikolay

**Comment by mikerodonnell89** [comment]
this has been fixed:
<img width="1302" alt="Screenshot 2024-11-22 at 11 36 16 AM" src="https://github.com/user-attachments/assets/530b52fb-f6ba-4ba3-b5b3-35ba0d5a95f6">

<img width="991" alt="Screenshot 2024-11-22 at 11 40 18 AM" src="https://github.com/user-attachments/assets/ce84af00-8713-42ce-863d-6fa41d41fde2">
